contents/docs/experiments/analyzing-results.mdx

```diff
@@ -0,0 +1,163 @@
+export const DeltaChartHoverLight = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/popup_light_8fe379428f.png"
+export const DeltaChartHoverDark = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/popup_dark_cd122710e7.png"
+export const FunnelBreakdownLight = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/funnel_light_e933e07d24.png"
+export const FunnelBreakdownDark = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/funnel_dark_c6a7c3e830.png"
+
+Once your experiment has been running and gathering data, you'll want to analyze the results to determine if your changes had a statistically significant impact. This guide walks you through understanding and interpreting experiment results in PostHog.
+
+## Understanding the results page
+
+### Exposures panel
+
+The exposures panel shows you how many users have been exposed to each variant of your experiment:
+
+<ProductScreenshot
+    imageLight={ExposuresPanelLight} 
+    imageDark={ExposuresPanelDark}
+    alt="Experiment exposures panel showing user distribution across variants" 
+    classes="rounded"
+/>
+
+This collapsible panel displays:
+- **Exposure count**: The total number of unique users who have seen each variant
+- **Percentage distribution**: How users are split between variants (typically 50/50 for two variants)
+
+The exposure numbers help you verify that:
+1. Your experiment is running and users are being assigned to variants
+2. The split between variants matches your expected allocation
+3. You have sufficient sample size for meaningful results
+
+### Metric results and significance
+
+For each metric you're tracking, PostHog displays a delta chart showing the performance of each variant compared to the control:
+
+<ProductScreenshot
+    imageLight={DeltaChartLight} 
+    imageDark={DeltaChartDark}
+    alt="Delta chart showing experiment results with confidence intervals" 
+    classes="rounded"
+/>
+
+## How to determine if a variant is significant
+
+The most important question when analyzing results is: **"Is my variant statistically significant?"**
+
+Look for these clear visual indicators in the delta chart:
+
+1. **Color-coded deltas**: When a variant achieves statistical significance:
+   - **Green with ↑ arrow**: The variant is winning (positive significant change)
+   - **Red with ↓ arrow**: The variant is losing (negative significant change)
+   - **No color**: The result is not statistically significant
+
+2. **Confidence/Credible intervals**: The horizontal bars show the 95% confidence interval (frequentist) or credible interval (Bayesian):
+   - If the interval doesn't cross zero, the result is statistically significant
+   - The narrower the interval, the more precise your estimate
+
+3. **Hover details**: When you hover over a variant in the delta chart, you'll see detailed statistics:
+
```

Comment:
Do we have resources to point people to who want a quick refresher on basic stats? :P